### PR TITLE
refactor(schemas): remove refs of invitation's schemas

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

**Review by spaenleh — APPROVED** [review]
I have nothing special to say, looks good 🛴

File: src/services/item/test/fixtures/items.ts [hunk 1]
```diff
@@ -168,12 +170,16 @@ export class ItemTestUtils {
...
+    const { item, permission, creator, parentItem } = options;
+    let member = options.member;
     const newItem = await this.saveItem({
       item,
       actor: creator ?? member,
       parentItem,
     });
+    if (!member) {
+      member = await saveMember();
+    }
```
Comment (ReidyT) [inline src/services/item/test/fixtures/items.ts:182]: Why should we save the member if not null? Also, do you know why the member can be undefined? I know it's not you who initially allowed undefined member...

Comment (ReidyT) [reply src/services/item/test/fixtures/items.ts:182]: Maybe it's just because the actor type in the tests is `Member | null`.

**Review by ReidyT — APPROVED** [review]
LGTM I just let one question 🫠

File: src/services/item/test/fixtures/items.ts (hunk 1 above)
Comment (Defferrard) [reply src/services/item/test/fixtures/items.ts:182]: I've created a use for that :)

**Pull request merged**